daemon/cluster/executor/container/container.go

```diff
@@ -374,6 +376,12 @@ func (c *containerConfig) hostConfig() *enginecontainer.HostConfig {
 		}
 	}
 
+	labels := c.task.Networks[0].Network.Spec.Annotations.Labels
+	name := c.task.Networks[0].Network.Spec.Annotations.Name
+
+	if v, ok := labels[naming.PredefinedLabel]; ok && v == "true" && name == "host" {
+		hc.NetworkMode = "host"
```

Comment:
No, this is required only for `host`, because `buildSandboxOption` is the one which checks that field to add the `host` specific option for the default sandbox.